analyzer/uncategorized/helpers.go

```diff
@@ -1,18 +1,19 @@
 package common
 
 import (
-	"fmt"
 	"io"
+
+	"github.com/oasisprotocol/oasis-indexer/log"
 )
 
-func CloseOrLog(c io.Closer) {
+func CloseOrLog(c io.Closer, logger *log.Logger) {
```

Comment:
This and `WriteOrLog` don't seem to be used anywhere. Can we remove them? I don't think the indexer will be writing to files much, or at all. It's all the DB.

Reply:
they're currently not. it would be nice though. we have some places where we `defer xxx.Close()` and ignore errors, e.g. SendBatch with `batchResults`.

in nexus I ended up using these in the API server. WriteOrLog in particular appeared a lot in the handlers, and indexer is similar

Reply:
#184 :confetti_ball: